Library/Formula/gst-plugins-good.rb

```diff
@@ -50,6 +50,15 @@ class GstPluginsGood < Formula
 
   depends_on "libogg" if build.with? "flac"
 
+  if MacOS.version < :lion
+    # Snow Leopard and below don't have strnlen()
+    # https://bugzilla.gnome.org/show_bug.cgi?id=756154
+    patch do
+      url "http://cgit.freedesktop.org/gstreamer/gst-plugins-good/patch/?id=fc203a4bd7eb1cecc0e17bcb7ec67e0672806867"
```

Comment:
Could you possibly submit this to https://github.com/Homebrew/patches and use that URL? We've found `cgit` URLs tend to churn the checksum every now and again and then we have issues verifying with upstream the change wasn't malicious.